Lib/test/support/__init__.py

```diff
@@ -2873,3 +2875,11 @@ def __iter__(self):
         if self.iter_raises:
             1/0
         return self
+
+
+def in_systemd_nspawn() -> bool:
```

Comment:
Please explain a little bit what is nspawn and what is the purpose of this function in a docstring or a comment.